src/reference/supported-platforms.md

```diff
@@ -40,10 +40,10 @@ following table:
 |Platform version|Supported|Best effort|Unsupported|
 |----------------|---------|-----------|-----------|
 | Android SDK    |21-34|19-20|18-|
-| iOS            |16|11-15|10-, arm7v 32-bit|
+| iOS            |16|11-15, 17|10-, arm7v 32-bit|
 | Linux Debian   |10-12|9-|any 32-bit|
-| Linux Ubuntu   |18.04 LTS|20.04-23.04|any 32-bit|
-| macOS          |Ventura (13+)|Mojave (10.14) to Monterey (12)|High Sierra (10.13-) |
+| Linux Ubuntu   |20.04 LTS|20.04-23.04|any 32-bit|
```

Comment:
This is confusing, 20.04 is both Supported and Best Effort, while 18.04 is now neither supported, nor best effort, or even unsupported.

Reply:
Oops, the best effort was supposed to be updated to `20.10-23.04`.

Reply:
As for 18.04, it has reached its end of its standard LTS support. Do you think it's worth adding to the `Unsupported` column or do you think it can be gleamed from not being in the first two columns?